index.bs

```diff
@@ -7119,9 +7121,16 @@ To <dfn>get the realm info</dfn> given |environment settings|:
 
     <dt>|global object| is a {{DedicatedWorkerGlobalScope}} object
     <dd>
+      1. Let |owners| be an empty [=/list=].
+      1. For each |owner| in the |global object|'s associated [=owner set=]:
+         1. Let |owner environment settings| be the [=environment settings object=] whose
+            [=relevant global object=] is |owner|.
+         1. Let |owner realm info| be the result of [=get the realm info=] given |owner environment settings|.
+         1. Append |owner realm info| to |owners|.
```

Comment:
Do we want all the owner realm info, or just the realm id? In `getRealms` providing all the information is going to lead to duplicate entries for the owners, and the CDDL suggests that you just wanted the id.

Reply:
Indeed, we just want the ID.